format/view-spec.md

```diff
@@ -160,6 +179,56 @@ Each entry in `version-log` is a struct with the following fields:
 | _required_  | `timestamp-ms` | Timestamp when the view's `current-version-id` was updated (ms from epoch) |
 | _required_  | `version-id`   | ID that `current-version-id` was set to |
 
+#### Table Identifier
+
+The table identifier holds a reference, containing a namespace and a name, of a table or view in the catalog.
```

Comment:
maybe simplified as
```
The table identifier for the storage table that stores the precomputed results.
```

also `the catalog` is probably not the most accurate, as it can be interpreted as the same catalog as the view definition.